Library/Formula/iojs.rb

```diff
@@ -0,0 +1,134 @@
+# Note that x.even are stable releases, x.odd are devel releases
+class Iojs < Formula
+  homepage "https://iojs.org/"
+  url "https://iojs.org/dist/v1.0.1/iojs-v1.0.1.tar.gz"
+  sha256 "c26052dad5d9ccd8a7b9134806994ebf2d217a57d6efd5633e01c5cc1dcdedc5"
+
+  conflicts_with "node", :because => "io.js includes a symlink named node for compatibility."
+
+  bottle do
+    revision 1
+    sha1 "a98a1df66cfb0712b14489186c46f7087ba35bd7" => :yosemite
+    sha1 "0cd45412840a67d5d65e6bc3c0c3bcf8bc23153c" => :mavericks
+    sha1 "977332381c033626b991002c27e738c144ebbaac" => :mountain_lion
+  end
+
+  deprecated_option "enable-debug" => "with-debug"
```

Comment:
It's a new formula, we don't need to deprecate the option since it hasn't technically existed yet :)